packages/soci-snapshotter/Cargo.toml

```diff
@@ -12,15 +12,17 @@ path = "../packages.rs"
 releases-url = "https://github.com/awslabs/soci-snapshotter/releases"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/awslabs/soci-snapshotter/archive/refs/tags/v0.9.0.tar.gz"
-sha512 = "3f196e20d300ecb585b57810caa19fd7dd715a7215ba2e7e3c49d769bd662b65c40c291e7cd782b53e0203bbbfd20b0238bcf38bf67b19c0fe350ac14843c53c"
-bundle-root-path = "soci-snapshotter-0.9.0/cmd"
+url = "https://github.com/awslabs/soci-snapshotter/archive/refs/tags/v0.11.1.tar.gz"
```

Comment:
You should use this URL:

https://github.com/awslabs/soci-snapshotter/archive/v0.11.1/soci-snapshotter-0.11.1.tar.gz

That's what is used in other packages.